account_banking_pain_base/models/res_company.py

```diff
@@ -22,7 +22,6 @@ class ResCompany(models.Model):
         help="This will be used as the 'Initiating Party Identifier' in "
         "the PAIN files generated by Odoo.")
 
-    @api.one
```

Comment:
It is ok to remove `@api.one` here, but mind that `self` is not supposed to be a singleton.

In my opinion you should add a `self.ensure_one()` in `method _default_initiating_party()`, and then, in file `post_install.py`, ensure that the method is invoked separately for each company. That means, replace:

```
companies._default_initiating_party()
```

with:

```
for company in companies:
    company._default_initiating_party()
```